index.html

```diff
@@ -473,15 +474,39 @@ <h4>Diana Pertersen</h4>
     <div class="container">
       <div class="row">
         <div class="col-lg-12 text-center">
-          <h2 class="section-heading text-uppercase">We'd love to hear from you</h2>
-          <h3 class="section-subheading text-muted">If you want to know more about our products, features, pricing or anything at all, we are here to answer your questions</h3>
+          <h2 class="section-heading">We'd love to hear from you</h2>
+          <h3 class="section-subheading text-white-50">If you want to know more about our products, features, pricing or anything at all, we are here to answer your questions</h3>
         </div>
       </div>
       <div class="row">
         <div class="col-lg-12">
-          <form id="contactForm" name="sentMessage" novalidate="novalidate">
             <div class="row">
               <div class="col-md-6">
+			  <p class="h2 text-white">Office Locations</p>
+			  </br>
```

Comment:
Don't use `<br>`.

Use [Bootstrap Spacing](https://getbootstrap.com/docs/4.2/utilities/spacing/) instead?
Maybe `mb-3` or `mb-4`?

Reply:
okay. Got it!